scripts/delete_db_and_setup.sh

```diff
@@ -0,0 +1,4 @@
+rm backend/run/db.sqlite3
+python3 backend/manage.py migrate
+python3 backend/manage.py createsuperuser --email admin@exampleee.com --username admin
```

Comment:
I wouldn't use a predefined mail address, let the user enter one
```suggestion
python3 backend/manage.py createsuperuser  --username admin
```